exercises/concept/high-school-sweetheart/.docs/introduction.md

```diff
@@ -42,11 +42,17 @@ Elixir provides many functions for working with strings in the `String` module.
 
 ## Pipe Operator
 
-The `|>` operator is called the pipe operator. It can be used to chain function calls together in such a way that the value returned by the previous function call is passed as the first argument to the next function call.
+The `|>` operator is called the pipe operator. It is used to chain function calls together in such a way that the value returned by the previous function call is passed as the first argument to the next function call.
```

Comment:
I like this adjustment, the original language insinuates that `|>` is but one of the ways to chain functions together, but I don't believe there is another way to really do so.

This would also need to be updated in [pipe-operator/introduction.md](https://github.com/exercism/elixir/blob/main/concepts/pipe-operator/introduction.md).

Reply:
Done ✅